packages/cli/src/utils/__tests__/error-handler.test.ts

```
@@ -29,6 +29,16 @@ describe("error handler and formatter", () => {
     expect(errorSpy).toHaveBeenCalledWith("❌ bad input");
   });
 
+  it("prints dry-run hint for validation errors", async () => {
+    const logSpy = vi.spyOn(console, "log").mockImplementation(() => undefined);
+
+    await handleCommandAction(async () => {
+      throw new CLIError("Invalid JSON input. Please provide a valid JSON string to --input.", ExitCode.VALIDATION_ERROR);
```

ESLint

[error] Replace `"Invalid·JSON·input.·Please·provide·a·valid·JSON·string·to·--input.",·ExitCode.VALIDATION_ERROR` with `⏎········"Invalid·JSON·input.·Please·provide·a·valid·JSON·string·to·--input.",⏎········ExitCode.VALIDATION_ERROR⏎······`

(prettier/prettier)